cypress/steps/yasgui-steps.ts

```diff
@@ -6,4 +6,11 @@ export class YasguiSteps {
     static getTabs() {
         return cy.get('.tab');
     }
+   static executeQuery() {
```

Comment:
Execute query method should be in the yasqe steps and I believe I moved it there

Reply:
This function is doubled after rebase. I will remove this one.